src/Loader.php

```diff
@@ -294,61 +294,63 @@ public static function register_scripts() {
 			return;
 		}
 
+		$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+
 		wp_register_script(
 			'wc-csv',
-			self::get_url( 'csv-export/index.js' ),
+			self::get_url( 'csv-export/index' . $suffix . '.js' ),
```

Comment:
Might be kind of cool if `get_url()` appended the suffix automatically or based on argument, but may be more trouble than its worth.

Reply:
Good idea - this also highlighted that I missed adding this suffix to the `wp-admin-scripts` directory!

Reply:
Done in 86926aed.